tests/approaches/test_interactive_approach.py

```diff
@@ -52,38 +44,3 @@ def test_interactive_learning_approach():
             pass
         # We won't check the policy here because we don't want unit tests to
         # have to train very good models, since that would be slow.
-
-    # Test teacher
-    traj = dataset.trajectories[0]
-    for s in traj.states:
-        ground_atoms = sorted(utils.abstract(s, env.predicates))
-        for g in ground_atoms:
-            assert approach.ask_teacher(s, g)
-
-
-def test_interactive_learning_approach_no_ground_atoms():
```

Comment:
I'm removing this test because the expected behavior is different now; the agent will not attempt to learn about a predicate that it has no data about, because it won't even know that this predicate exists.